### PR TITLE
chore: increase coverage

### PR DESCRIPTION
**Pull request by craigbilner**
We don't need the old way of mocking with `jest-configurator` plus some components were coverage checking unrun code

chore: ignore coverage json when prettying
chore: remove flags from yarn in CLI
chore: remove unecessary mocks
chore: jest-configurated packages that need platform specific tests

### CODE REVIEW

File: packages/pagination/__tests__/android/pagination-wrapper.native.test.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import tests from "../test-wrapper-helper";
+import pagination from "../../pagination-wrapper";
+
+describe("Pagination Wrapper Native", tests(pagination));
```
Comment (colincclark) [inline packages/pagination/__tests__/android/pagination-wrapper.native.test.js]: is it worth explicitly stating this is for Android in this describe?

File: packages/pagination/__tests__/ios/pagination-icons.native.test.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import tests from "../test-icons-helper";
+
+describe("PaginationIcons Native", tests);
```
Comment (colincclark) [inline packages/pagination/__tests__/ios/pagination-icons.native.test.js]: Same question re: explicitly stating iOS here in the describe...

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/3d3b38a27d80b1cb228012455ac6949885db7a56/dextrose-test/index.html

File: packages/caption/__tests__/android/caption.android.test.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import "react-native";
+import shared from "../shared";
+
+shared();
```
Comment (jimhunty) [inline packages/caption/__tests__/android/caption.android.test.js]: Wrap in a describe?

File: packages/caption/__tests__/ios/caption.ios.test.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+import "react-native";
+import shared from "../shared";
+
+shared();
```
Comment (jimhunty) [inline packages/caption/__tests__/ios/caption.ios.test.js]: Wrap in a describe?

File: packages/caption/__tests__/web/caption.web.test.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+import "react-native";
+import shared from "../shared";
+
+shared();
```
Comment (jimhunty) [inline packages/caption/__tests__/web/caption.web.test.js]: Wrap in a describe?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/eb529a68d8f7a98686fa3ebac20431d970cbc66e/dextrose-test/index.html

**Pull request merged**